tools/clang/test/CodeGenDXIL/hlsl/objects/NodeObjects/mesh-node-multi-setoutcounts.hlsl

```diff
@@ -0,0 +1,46 @@
+// RUN: not %dxc -T lib_6_9 %s 2>&1 | FileCheck %s
+
+// REQUIRES: dxil-1-9
+
+// Ensure multiple setMeshOutputCounts produce a validation error
```

Comment:
I think validation tests should be implemented as IR tests, shouldn't they?  Presumably, we would eventually want Sema or something else to catch some of these errors before validation if possible, which would break any validation test that relies on compiling invalid HLSL.

`CodeGenDXIL` doesn't seem like the right place to put validator tests, and see previous comment about usage of the path `objects/NodeObjects`.

Reply:
Longer-term this may be the case, but is this important to fix for this experimental release?

Reply:
It's fine for this branch for now.